tests/CVTracker.cpp

```diff
@@ -73,10 +73,10 @@ TEST_CASE( "Track_Video", "[libopenshot][opencv][tracker]" )
     int height = ((float)fd.y2*360) - y;
 
     // Compare if tracked data is equal to pre-tested ones
-    CHECK(x >= 255); CHECK(x <= 257);
-    CHECK(y >= 133); CHECK(y <= 135);
-    CHECK(width >= 179); CHECK(width <= 181);
-    CHECK(height >= 165); CHECK(height <= 168);
+    CHECK(x == Approx(256).margin(1));
+    CHECK(y == Approx(134).margin(1));
+    CHECK(width == Approx(180).margin(1));
+    CHECK(height == Approx(166).margin(2));
```

Comment:
I wrote these ranges to approximate what the previous code was doing, but they really feel kind of broad. It seems like we should be able to be more precise than that, is there really that much variation in the results that we'll get?